src/commands/lambda/flare.ts

```diff
@@ -390,9 +453,20 @@ export const getLogStreamNames = async (cwlClient: CloudWatchLogsClient, logGrou
   const output: string[] = []
   for (const logStream of logStreams) {
     const logStreamName = logStream.logStreamName
-    if (logStreamName) {
-      output.push(logStreamName)
+    if (!logStreamName) {
+      continue
     }
+    if (rangeSpecified) {
+      const lastEventTime = logStream.lastEventTimestamp
+      const firstEventTime = logStream.firstEventTimestamp
+      if (!lastEventTime || !firstEventTime) {
+        continue
+      }
```

Comment:
Good point. According to the documentation, the timestamps can take up to an hour to update. So a very recently executed log could return undefined for these values. 
I think it makes more sense to add these streams when the values are undefined instead of continuing.
Updated the code to:
```
      if (lastEventTime && firstEventTime && (lastEventTime < startMillis || firstEventTime > endMillis)) {
        continue
      }
```